spec/cb/utils/validator_spec.rb

```diff
@@ -12,127 +12,131 @@
 
 module Cb
   describe ResponseValidator do
-    let(:response)          { double(HTTParty::Response) }
-    let(:response_property) { double('HTTParty::Response.response') }
+    let(:response)          { double(HTTParty::Response, response: response_property, code: response_code) }
+    let(:response_property) { double('HTTParty::Response.response', body: response_body) }
+    let(:response_body)     { nil }
+    let(:response_code)     { 200 }
 
-    before :each do
-      allow(response).to receive(:response).and_return(response_property)
-      allow(response).to receive(:code).and_return(200)
-    end
+    subject(:validation) { ResponseValidator.validate(response) }
```

Comment:
@oochelz I used a named subject since it still cuts down on code, but doesn't require as much change, while being more explicit than "subject" littered throughout code.